crowbar_framework/app/models/neutron_service.rb

```diff
@@ -15,6 +15,8 @@
 # limitations under the License.
 #
 
+require_relative '../helpers/barclamp/neutron_helper.rb'
```

Comment:
Done in https://github.com/crowbar/barclamp-neutron/commit/2e545420df705c6ae99da591c1280414647aa26e